source/_components/delijn.markdown

```diff
@@ -0,0 +1,72 @@
+---
+title: "De Lijn"
+description: "Instructions on how to integrate De Lijn (Flemish public transport company) departure times into Home Assistant."
+ha_release: 0.97
+ha_category: 
+  - Transport
+  - Sensor
+ha_iot_class: Cloud Polling
+---
```

Comment:
Should we add a logo?

Reply:
yes we should 😄

Reply:
Can I add it via github directly? I don't have a local version of the repo on my pc and I think I read you could not add a picture via github. If possible, please let me know how. Otherwise, if you could add them yourself, these are some that I found:
- the logo on their site: https://static.delijn.be/Images/dl-logo_tcm3-1040.svg
- the logo on Wikipedia: https://nl.wikipedia.org/wiki/De_Lijn_(vervoermaatschappij)#/media/Bestand:De_Lijn.svg
- a vector version of their logo (although the blue seems a little bit off): https://seeklogo.com/vector-logo/39175/de-lijn

Hope it helps. Cheers!

Reply:
Done.